# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1452348168 -3600
#      Sat Jan 09 15:02:48 2016 +0100
# Node ID 5c5af828c248794bc94c0416e28740e6f5002ff3
# Parent  4e2e93ec558a71971c24d01b715b06ec2e866d71
gui: only center line with debugger pointer if line is at the top or bottom

* file-editor-tab.cc (insert_debugger_pointer): use center_current_line with new
  flag for only centering if the line is at the top or the bottom

diff --git a/libgui/src/m-editor/file-editor-tab.cc b/libgui/src/m-editor/file-editor-tab.cc
--- a/libgui/src/m-editor/file-editor-tab.cc
+++ b/libgui/src/m-editor/file-editor-tab.cc
@@ -2203,17 +2203,17 @@ file_editor_tab::insert_debugger_pointer
       else
         dp = new marker (_edit_area, line, marker::debugger_position);
 
       connect (this, SIGNAL (remove_position_via_debugger_linenr (int)),
                dp,   SLOT (handle_remove_via_original_linenr (int)));
       connect (this, SIGNAL (remove_all_positions (void)),
                dp,   SLOT (handle_remove (void)));
 
-      center_current_line ();
+      center_current_line (false);
     }
 }
 
 void
 file_editor_tab::delete_debugger_pointer (const QWidget *ID, int line)
 {
   if (ID != this || ID == 0)
     return;
